packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -138,6 +138,7 @@ export const handler = capture.wrapHandler(async (event: SQSEvent) => {
     try {
       log(`Body: ${message.body}`);
       const { s3BucketName, s3FileName, documentExtension } = parseBody(message.body);
+      capture.setExtra({ s3FileName });
```

Comment:
Noticed we were not sending the file name to Sentry - this should make it easier to get it instead of relying on breadcrumbs/logs